packages/nomad/template-manager.hcl

```diff
@@ -1,126 +1,66 @@
-variable "gcp_zone" {
-  type    = string
-}
-
-variable "gcp_project" {
-  type    = string
-}
-
-variable "gcp_region" {
-  type    = string
-}
-
-variable "port" {
-  type    = number
-  default = 5009
-}
-
-variable "docker_registry" {
-  type    = string
-  default = ""
-}
-
-variable "api_secret" {
-  type    = string
-  default = ""
-}
-
-variable "otel_tracing_print" {
-  type    = string
-  default = ""
-}
-
-variable "environment" {
-  type    = string
-  default = ""
-}
-
-variable "template_manager_checksum" {
-  type    = string
-  default = ""
-}
-
-variable "google_service_account_key" {
-  type    = string
-  default = ""
-}
-
-variable "bucket_name" {
-  type    = string
-  default = ""
-}
-
-variable "template_bucket_name" {
-  type    = string
-  default = ""
-}
-
-variable "otel_collector_grpc_endpoint" {
-  type    = string
-  default = ""
-}
-
-variable "logs_collector_address" {
-  type    = string
-  default = ""
-}
-
 job "template-manager" {
-  datacenters = [var.gcp_zone]
+  datacenters = ["${gcp_zone}"]
   node_pool  = "build"
   priority = 70
 
   group "template-manager" {
     network {
       port "template-manager" {
-        static = var.port
+        static = "${port}"
       }
     }
 
     service {
       name = "template-manager"
-      port = var.port
+      port = "${port}"
 
       check {
         type         = "grpc"
         name         = "health"
         interval     = "20s"
         timeout      = "5s"
         grpc_use_tls = false
-        port         = var.port
+        port         = "${port}"
       }
     }
 
     task "start" {
       driver = "raw_exec"
 
+%{ if update_stanza }
+      # If we need more than 30s we will need to update the max_kill_timeout in nomad
+      # https://developer.hashicorp.com/nomad/docs/configuration/client#max_kill_timeout
+      kill_timeout = "10m"
```

Comment:
there is maximum kill timeout 10m for cluster now. I guess in some cases for build finish we need more

Reply:
done